src/js/modules/info.js

```diff
@@ -9,7 +9,7 @@ function getPublicationDate(rawDate) {
     return '';
   }
   var date = new Date(rawDate);
-  return '<p>Veröffentlicht am: ' + date.getDate() + '.' + (date.getMonth() + 1) + '.' + date.getFullYear() + '</p>';
+  return '<p>' + $.i18n( 'tab_info_title_published' ) + ': ' + date.getDate() + '.' + (date.getMonth() + 1) + '.' + date.getFullYear() + '</p>';
```

Comment:
Can we abstract the date format into variables here for different locales? Like i.e. for English `month, dd, yyyy` or `mm/dd/yyyy`?